packages/devtools_app/test/matchers/matchers.dart

```diff
@@ -22,8 +20,8 @@ RemoteDiagnosticsNode findNodeMatching(
   if (node.childrenNow == null) {
     return null;
   }
-  for (var child in node.childrenNow) {
-    final match = findNodeMatching(child, text);
+  for (var child in node.childrenNow!) {
+    final match = findNodeMatching(child!, text);
```

Comment:
child shouldn't be nullable. childrenNow should be a non nullable list of non nullable elements

Reply:
merged